shared/actions/teams.tsx

```diff
@@ -147,6 +147,12 @@ const leaveTeam = async (action: TeamsGen.LeaveTeamPayload, logger: Saga.SagaLog
 
 const leftTeam = () => RouteTreeGen.createNavUpToScreen({routeName: 'teamsRoot'})
 
+const loadWelcomeMessage = async (action: TeamsGen.LoadWelcomeMessagePayload, _: Saga.SagaLogger) => {
+  const {teamID} = action.payload
+  const message = await RPCChatTypes.localGetWelcomeMessageRpcPromise({teamID})
```

Comment:
this'll not typically error out right?

Reply:
It might. It ends up loading some chat messages.

Reply:
left as-is it'll black bar. we can keep that or catch and do something else

Reply:
ok, what should it do?